src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -104,8 +104,10 @@ private void addRandomUsers() {
     Collections.shuffle(randomUsernames);
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
-      User user = new User(UUID.randomUUID(), randomUsernames.get(i), BCrypt.hashpw("password", BCrypt.gensalt()),
-          Instant.now());
+      User.Builder userBuilder = new User.Builder(UUID.randomUUID(), randomUsernames.get(i), BCrypt.hashpw("password", BCrypt.gensalt()), Instant.now());
+      userBuilder.setBio("I am a User on the Team TNT Chat App.");
```

Comment:
Would be good to make this a static final class variable just in case we want to use it elsewhere in DefaultDataStore. It's also a bit cleaner:

private static final DEFAULT_BIO = "....."

Same with the default picture url:

private static final DEFAULT_PICTURE_URL = "...."